pkg/provisioner/ironic/ironic.go

```diff
@@ -114,10 +117,22 @@ func init() {
 	if ironicTrustedCAFile == "" {
 		ironicTrustedCAFile = "/opt/metal3/certs/ca/crt"
 	}
+	ironicClientCertFile = os.Getenv("IRONIC_CLIENT_CERT_FILE")
+	if ironicClientCertFile == "" {
+		ironicClientCertFile = "/opt/metal3/certs/client/crt"
+	}
+	ironicClientPrivKeyFile = os.Getenv("IRONIC_CLIENT_PRIVATE_KEY_FILE")
+	if ironicClientPrivKeyFile == "" {
+		ironicClientPrivKeyFile = "/opt/metal3/certs/client/key"
+	}
 	ironicInsecureStr := os.Getenv("IRONIC_INSECURE")
 	if strings.ToLower(ironicInsecureStr) == "true" {
 		ironicInsecure = true
 	}
+	ironicSkipClientVerifyStr := os.Getenv("IRONIC_SKIP_CLIENT_VERIFY")
```

Comment:
```suggestion
	ironicSkipClientVerifyStr := os.Getenv("IRONIC_SKIP_CLIENT_SAN_VERIFY")
```
The variable name should provide more detail about the verification.

Reply:
Just in case, variable name was modeled after `IRONIC_INSECURE`, all related to `IRONIC_INSECURE` variables do not provide detail